index.js

```diff
@@ -79,6 +79,11 @@ async function main() {
     const DEFAULT_BRANCH = default_branch;
     core.debug(`DEFAULT_BRANCH: ${DEFAULT_BRANCH}`);
 
+    if (inputs.path) {
+      core.debug(`Changing directory to ${inputs.path}`);
+      await runShellCommand(`cd ${inputs.path}`);
```

Comment:
I don't think this is right, I'm pretty sure it will break the action, we check for changes using the path

https://github.com/gr2m/create-or-update-pull-request-action/blob/0eb61a2e8a7a26aa5dfd8c54fda5c10797fe9f29/index.js#L305

and add the changes using the path as well

https://github.com/gr2m/create-or-update-pull-request-action/blob/0eb61a2e8a7a26aa5dfd8c54fda5c10797fe9f29/index.js#L116-L119